perception/ground_segmentation/src/ray_ground_filter_nodelet.cpp

```diff
@@ -202,6 +169,7 @@ void RayGroundFilterComponent::ClassifyPointCloud(
       double local_max_slope = local_max_slope_;
       if (j == 0) {
         local_max_slope = initial_max_slope_;
+        prev_height = in_radial_ordered_clouds[i][j].point.z;
```

Comment:
Is it unnecessary if input_frame is base link?

Reply:
thanks, it is unnecessary. I will remove it.